Adds documentation for validating webhook changes (#1222)
kanisterio/kanister: docs/tooling.rst

    Passed the 'validation of phase deleteFromBlobStore in action delete' check.. ✅
    Passed the 'validation of phase restoreFromBlobStore in action restore' check.. ✅

> ``kanctl validate blueprint`` right now checks that kanister function names
> are correct and all the mandatory arguments to the functions are provided.

Reviewer: ```suggestion
``kanctl validate blueprint`` currently verifies the Kanister function names
and the presence of the mandatory arguments to those functions.
```
